zulip_api/setup.py

```diff
@@ -59,28 +59,22 @@ def recur_expand(target_root, dir):
     entry_points={
         'console_scripts': [
             'zulip-send=zulip.send:main',
-            'zulip-bot-server=zulip.bot_server:main',
         ],
     },
-    test_suite='tests',
 )  # type: Dict[str, Any]
 
 setuptools_info = dict(
     install_requires=['requests>=0.12.1',
                       'simplejson',
                       'six',
                       'typing>=3.5.2.2',
-                      'flask>=0.12.2',
-                      'mock>=2.0.0',
-                      # for pep8 linter
-                      'pycodestyle==2.3.1',
```

Comment:
This dependency is for the pep8 linter and I figure we should move this code outside of both the packages soon, but perhaps not in this PR.